homeassistant/components/group.py

```diff
@@ -404,6 +404,9 @@ def _state_changed_listener(self, entity_id, old_state, new_state):
 
         This method must be run in the event loop.
         """
+        # remove
+        if new_state is None:
```

Comment:
This is not correct. Removing a state can trigger the group state to change. I think that instead we should check if the group is stopped or not to decide if we have to proceed